logstash-core/src/main/java/org/logstash/Timestamp.java

```diff
@@ -68,6 +68,11 @@ public String toString() {
         return iso8601Formatter.print(this.time);
     }
 
+    public long toEpochMilli() {
+        return (new Duration(JAN_1_1970.toDateTime(DateTimeZone.UTC), this.time).getMillis());
```

Comment:
Why not send [`org.joda.BaseDateTime#getMillis()`](http://joda-time.sourceforge.net/apidocs/org/joda/time/base/BaseDateTime.html#getMillis())?